homeassistant/components/vizio/__init__.py

```diff
@@ -24,13 +24,13 @@
 
 
 def validate_apps(config: ConfigType) -> ConfigType:
-    """Validate CONF_APPS is only used when CONF_DEVICE_CLASS == DEVICE_CLASS_TV."""
+    """Validate CONF_APPS is only used when CONF_DEVICE_CLASS is MediaPlayerDeviceClass.TV."""
     if (
         config.get(CONF_APPS) is not None
-        and config[CONF_DEVICE_CLASS] != DEVICE_CLASS_TV
+        and config[CONF_DEVICE_CLASS] is not MediaPlayerDeviceClass.TV
```

Comment:
Are we sure this will work? is `config[CONF_DEVICE_CLASS]` actually `MediaPlayerDeviceClass.TV` or the string?

Reply:
You're right. I'll fix that.